contrib/python-sambacc.spec

```diff
@@ -0,0 +1,66 @@
+%global bname sambacc
+# set xversion to define the default version number
+%define xversion 0.1
+# set pversion for a customized python package version string
+%{?!pversion: %define pversion %{xversion}}
+# set rversion for a customized rpm version
+%{?!rversion: %define rversion %{xversion}}
+
+
+Name:           python-%{bname}
```

Comment:
Why not `Name: python3-%{bname}` ?

Reply:
I hope its based on [Python Packaging Guidelines](https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_naming) from Fedora.

Reply:
Indeed. I think the idea is if that during the py2->py3 transition, and perhaps some future py3->py4 transition the source name is 'python-<blah>` but based on the build it gets named as python3-blah, python2-blah, etc.